### PR TITLE
Fix responsive layout of sections and lazy-load images

### PR DESCRIPTION
**Pull request by TimvdLippe**
Mostly this was setting `min-width` on items inside flex styled elements. Also the committee member images are now properly spacing on 2 levels. Lastly, lazy-load the images by first putting it in a `data-src` so that we are not loading every single image on the page at once.

### CODE REVIEW

**Comment by jeroentjuuhh** [comment]
Hi, I like your solution. But i think something when wrong with the min-height. It does not preserve ratio at chrome. Here is a screenshot;

image

I tried to fix it but I was not able to identify why it does not preserve the ratio...

**Comment by jeroentjuuhh** [comment]
### Problem
Okay so i did something stupid. Merged another branch into the master and had some ugly merge conflicts after that. I tried to fix them but was pretty difficult for me.

I tested everything but of course there could still have gone something wrong while merging.
Could you check it once more? Then we can merge this and go live with the improvements made!

**Comment by TimvdLippe** [comment]
@jeroentjuuhh Fixed the issues. Feel free to merge and upload to production!

File: style.css [hunk 1]
```diff
@@ -185,10 +200,18 @@ button:active {
...
+  .actual-photos > div {
+    width: 100%;
+  }
+  .actual-photos img {
+    margin: auto;
+  }
+}
+
 
 #program .content {
```
Comment (elarb) [inline style.css]: This one is in here twice

**Pull request merged**